R/hiper_glm.R

```diff
@@ -1,31 +1,104 @@
+
+####################### Helper functions ######################
+
+########## For cholesky ########
+## Find linear regression coefficients estimates via MLE,
+## with cholesky decomposition while solving matrices
+chol_finder <- function(design, outcome){
+  ## With least-square, we have,
```

Comment:
-1

Don't use comments to describe what is going on, try to make that clear from the code.

Use comments to describe why something is done.